usmqe_tests/api/ceph/ceph_test_import.py

```diff
@@ -0,0 +1,122 @@
+"""
+REST API test suite - ceph cluster import
+
+"""
+import pytest
+
+from usmqe.api.tendrlapi import cephapi
+
+
+LOGGER = pytest.get_logger('ceph_import_test', module=True)
```

Comment:
I would rename this file and logger to `test_cluster`. `usmqe_tests/api/ceph/ceph_test_import.py` is not valid for pytest testing.